content/docs/how_we_work/research-software-directory.md

```diff
@@ -0,0 +1,57 @@
+---
+# Page title as it appears in the navigation menu
+title: "Research Software Directory"
+# Adjust the weight to reorder menu items (lower numbers appear first)
+weight: 3
+---
+
+# Research Software Directory
+
+## About 
+
+The [Research Software Directory](https://research-software-directory.org/)(RSD) is a content management system created by the Netherlands eScience Center. 
+It aims to promote the visibility, impact and reuse of research software and to encourage proper citation to ensure researchers and RSEs get credit for their work.
+
+We (REG) have set up a page for [The Alan Turing Institute](https://research-software-directory.org/organisations/the-alan-turing-institute) on the RSD. 
+We hope that this page will be a place where we can showcase the variety of software the REG team have developed. This could be beneficial for ourselves, collaborators (past and present), and potential future hires - to get a feel for the kind of things the team/institute work on, or even spark new collaborations.
+
+As you can see, we've already added a few pieces of REG software to the Turing RSD. We're encouraging more people to do the same! 
+
+## Getting Started
+
+The RSD documentation can be found [here](https://research-software-directory.org/documentation/)
+
+### Logging in to the RSD
+
+You will need to use your [ORCID](https://orcid.org/) to log in to the RSD.
+
+To request access, you should send an email from your Turing email account to `rsd@esciencecenter.nl` stating the following information:
+
+- Your name.
+- Your affiliation(s) - i.e. The Alan Turing Institute
+- Your ORCID.
+- Any example links illustrating the content you wish to add.
+
+### How to join The Alan Turing Institute organisation on the RSD
+
+Once you have logged in to the RSD, you will need to send [one of the RSD admins](https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry) a message asking to be added to the Alan Turing Institute organisation.
```

Comment:
```suggestion
Once you have logged in to the RSD, you will need to send [one of the RSD admins](https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry) a message asking to be added to the The Alan Turing Institute organisation.
```

Reply:
Maybe we should add a test to CI to make sure that the 'The' is always capitalised?!

Reply:
Yes to capitalising the 'T', but we don't need 'the The'.

Reply:
will just fix this